src/common/transformations/include/transformations/common_optimizations/optimize_strided_slice.hpp

```diff
@@ -6,13 +6,13 @@
 
 #include <memory>
 #include <ngraph/opsets/opset1.hpp>
```

Comment:
please use ov headers here and below in this file if possible 
if ngraph headers are required, I guess we can move it to cpp file